### ISSUE TITLE
Is this the official repository for the winning distro of the hackathon?

### ISSUE THREAD

**Issue by revoltez** [MILOUDIAS/FennecOS#9]

**Comment by MILOUDIAS** [comment]
No, this is my personal distro made from scratch; I didn't use it during the hackathon, because my team used a different approach.

**Comment by revoltez** [comment]
@MILOUDIAS thanks, did your team win the hackathon? if so are there links to the official repo!

**Comment by MILOUDIAS** [comment]
Yeah, we won! When the distro is publicly released, I will send you the link here.
